openapi.yaml

```diff
@@ -77,6 +77,35 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ModelResponse"
+  /public/v1/quali-logos:
+    get:
+      tags:
+        - QualiLogo
+      description: Returns all quali logos
+      operationId: getAll
+      parameters:
+        - name: brand
+          in: query
+          required: false
+          schema:
+            $ref: "#/components/schemas/Brand"
+        - name: vehicleCategory
+          in: query
+          required: false
+          schema:
+            $ref: "#/components/schemas/VehicleCategory"
+        - name: language
+          in: query
+          required: false
+          schema:
+            $ref: "#/components/schemas/Language"
+      responses:
+        "200":
+          description: Qualilogos for a brand, vehicle category, language returned
```

Comment:
`Quali logos`
missing `.` at the end of the sentence

Reply:
Since none of the params are mandatory, im not sure we should word it like this. 
`Quali logos successfully returned.` ?